integration-tests/data/requests_2_20_0.json

```diff
@@ -1,7 +1,7 @@
 [
     {
         "package": "requests",
-        "version": "2.21.0",
+        "version": "2.20.0",
```

Comment:
reasons for downgrading this version?

Reply:
File version and data version were not matching. It was confusing, so correct the content and also relevant test cases.